src/days-2-4-adv/adv.py

```diff
@@ -71,6 +71,8 @@
             plyr.travel(cmd[0])
         elif cmd[0] == 'i' or cmd[0] == 'inventory':
             plyr.getInventory()
+        elif cmd[0] == 'score':
+            plyr.getScore()
```

Comment:
Looks good! I hope you get creative with the items and how they give points, it's what makes the game fun 😁